docs/howto/deployment/index.txt

```diff
@@ -3,8 +3,7 @@ Deploying Django
 
 Django's chock-full of shortcuts to make Web developer's lives easier, but all
 those tools are of no use if you can't easily deploy your sites. Since Django's
-inception, ease of deployment has been a major goal. There's a number of good
-ways to easily deploy Django:
+inception, ease of deployment has been a major goal.
```

Comment:
Just discussed it with @freakboy3742 . We're going to drop that sentence. Thanks.